### PR TITLE
Get Singularity CI tests working

### PR DESCRIPTION
**Pull request by Emiller88**
Should fix issues with singularity builds #145 and "fixes" the `bowtie*_build` tags not running.

### CODE REVIEW

File: tests/software/bowtie/align/test.yml [hunk 1]
```diff
@@ -42,6 +42,6 @@
     - path: output/bowtie/bowtie/NC_010473.rev.2.ebwt
       md5sum: f3c398bba5158f4039334a932d79c051
     - path: output/bowtie/test.out
-      md5sum: a23e9a2a76e949aeb3693bcfae41a615
+      md5sum: a81cb18024616415a6cec3108a36fccd
```
Comment (KevinMenden) [inline tests/software/bowtie/align/test.yml:45]: Any idea why this changed?

File: tests/software/bowtie/build/test.yml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-- name: bowtie build
+- name: bowtie build test
```
Comment (KevinMenden) [inline tests/software/bowtie/build/test.yml:1]: I thought we want to keep this minimal (without obvious 'test')

File: tests/software/bowtie2/build/test.yml [hunk 3]
```diff
@@ -1,4 +1,4 @@
-- name: bowtie2 build
+- name: bowtie2 build test
```
Comment (KevinMenden) [inline tests/software/bowtie2/build/test.yml:1]: same as above.

or is there some weird test-reason why we need this?

**Review by KevinMenden — COMMENTED** [review]
Looks good just a few questions basically.

The singularity version necessary I guess?

File: tests/software/bowtie2/build/test.yml (hunk 3 above)
Comment (drpatelh) [inline tests/software/bowtie2/build/test.yml:1]:
```suggestion
- name: bowtie2 build
```

File: tests/software/bowtie/build/test.yml (hunk 2 above)
Comment (drpatelh) [inline tests/software/bowtie/build/test.yml:1]:
```suggestion
- name: bowtie build
```

Comment (drpatelh) [reply tests/software/bowtie/build/test.yml]: Good spot 👍🏽

File: tests/software/bowtie/align/test.yml [hunk 4]
```diff
@@ -42,6 +42,6 @@
...
     - path: output/bowtie/test.bam
       should_exist: true
```
Comment (drpatelh) [inline tests/software/bowtie/align/test.yml:47]:
```suggestion
```

File: tests/software/bowtie/align/test.yml (hunk 1 above)
Comment (drpatelh) [reply tests/software/bowtie/align/test.yml:45]:
```
# reads processed: 10000
# reads with at least one alignment: 2 (0.02%)
# reads that failed to align: 9998 (99.98%)
Reported 2 alignments
```

This is the latest version which is only showing 2 alignments...are you able to run `pytest` via the old tests so we can see the difference?

**Review by drpatelh — APPROVED** [review]
LGTM! I think we should create a separate issue for why we had to change the `md5sum` for `bowtie align` and come back to that so we can get this in to start figuring out how to deal with Conda!

File: tests/software/bowtie/align/test.yml (hunk 1 above)
Comment (drpatelh) [reply tests/software/bowtie/align/test.yml:45]: The old version generated the output below which is why the tests are failing:

```
# reads processed: 10000
# reads with at least one alignment: 1 (0.01%)
# reads that failed to align: 9999 (99.99%)
Reported 1 paired-end alignments
```

I guess this highlights the importance of using seed values where possible in `main.nf` for the tests to always generate the same alignments and not by random. At least, this is what I think is happening here.

**Comment by drpatelh** [comment]
💥

**Pull request merged**